test/test_images/performance/aggregator.go

```diff
@@ -56,12 +56,16 @@ type aggregatorExecutor struct {
 	// GRPC server
 	listener net.Listener
 	server   *grpc.Server
+
+	// Mako additional tags
+	additionalTags []string
```

Comment:
The comment would be unnecessary if the name of the variable reflected the usage better (e.g. `makoTags` or similar)